webonary-cloud-api/lambda/db.ts

```diff
@@ -63,8 +63,122 @@ export interface DbFindParameters {
 
 export const DB_NAME = process.env.DB_NAME as string;
 
+export const DB_MAX_DOCUMENTS_PER_CALL = 100;
+export const DB_MAX_UPDATES_PER_CALL = 50;
+
 // Vietnamese seem to have the most Latin diacritics, so use this for case and diacritic insensitive search
```

Comment:
seem -> seems,
Now that I'm thinking on this I'm surprised there isn't a better way to do case and diacritic insensitive searching...

Reply:
case insensitive search can be done by regex, but handling diacritical markers in regex is not so easy nor comprehensive. i think we might need to do this, though, once we get more user data and feedback